### PR TITLE
Eth address in TaskToCompute

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #150

### CODE REVIEW

File: tests/message/test_tasks.py [hunk 1]
```diff
@@ -115,3 +121,13 @@ def test_concent_enabled_default_true(self):
...
+
+    def test_ethereum_address(self):
+        msg = factories.TaskToComputeFactory()
+        serialized = shortcuts.dump(msg, None, None)
+        msg_l = shortcuts.load(serialized, None, None)
+        for addr_slot in (
+                'requestor_ethereum_address',
+                'provider_ethereum_address'):
+            address = getattr(msg_l, addr_slot)
+            self.assertEqual(len(address), 2 + (20*2))
```
Comment (rwrzesien) [inline tests/message/test_tasks.py]: Wouldn't it be worth to compare `requestor_ethereum_public_key` and `provider_ethereum_public_key` before and after serialization ?

Comment (jiivan) [reply tests/message/test_tasks.py]: Actually it's already done in `SerializationMixin`.

File: golem_messages/message/tasks.py [hunk 2]
```diff
@@ -89,6 +91,14 @@ def __init__(self, header: datastructures.MessageHeader = None,
         if self.concent_enabled is None:
             self.concent_enabled = True
 
+    @property
+    def requestor_ethereum_address(self):
+        return '0x{}'.format(self.requestor_ethereum_public_key[:20].hex())
```
Comment (Krigpl) [inline golem_messages/message/tasks.py]: This isn't correct. The right formula is `address = sha3(public_key)[12:]`. See `pubkeytoaddr` in `golem.utils`

**Comment by Krigpl** [comment]
Actually, it is really needed? Since we can easily compute it from the public key, so we can pass only the public key.

**Comment by jiivan** [comment]
It's contrary to the documentation, so I believe it will prove useful.

**Pull request merged**